src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -172,7 +236,7 @@ const styles = StyleSheet.create({
     paddingTop: dimen.normalizeDimen(12) + top,
     zIndex: 10
   }),
-  Header: {
+  headerContainer: {
     width: '100%',
     flexDirection: 'row',
     height: dimen.size.TOPIC_FEED_HEADER_HEIGHT,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style for the headerContainer has been updated. Make sure this change doesn't break the layout or positioning of elements.
```diff
-  Header: {
+  headerContainer: {
     width: '100%',
     flexDirection: 'row',
     height: dimen.size.TOPIC_FEED_HEADER_HEIGHT,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->